.github/workflows/limit-complexity.yml

```diff
@@ -0,0 +1,16 @@
+name: Limit complexity
+
+on: [push, pull_request]
+
+jobs:
+  complexcheck:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install lizard
+      run: pip install lizard
+
+    - name: Limit complexity
+      run: $HOME/.local/bin/lizard --exclude "./test/*" --CCN 5
```

Comment:
The max CCN is 5 here, but mentioned as 3 in the README?

Reply:
OK got it initiated with 5 now updated to 5